### ISSUE TITLE
No longer working :(

### ISSUE THREAD

**Issue by Ang41187** [mchristopher/PokemonGo-DesktopMap#1356]
Well it was nice while it lasted, but after the latest update the map appears to be broke. Not seeing any pokemon anymore. It'd be great if there was an update, but I'm not holding my breath since it's been so long.

**Comment by Rayenous** [comment]
....Mine is still working.

**Comment by Ang41187** [comment]
You're lucky @Rayenous. Even after completely removing the app, doing a fresh reinstall and creating a new PTC account and API key I'm still only getting pokestops and gyms to pop up. I'm kinda in the middle of nowhere, so this app was very helpful in my little town.

**Comment by manolomanila** [comment]
You might be doing something wrong, maybe the accounts have captcha or are banned. Mine is still working and I recently installed it on another PC and works as well.

**Comment by Rayenous** [comment]
Something I noticed recently is that using a default config.ini (or no config.ini) file will allow each account to scan once... subsequent scans will then only show stops (I'm not scanning close enough to gyms to know those are being scanned)

In my current config.ini, I have set the scan speed to 44 seconds... which seems high, but I am scanning a relatively small area with a relatively large number of accounts.  -  Whole area is still scanned fast.

TL;DR:  Try editing the scan speed in your config.ini 
Info on this can be found here: https://github.com/mchristopher/PokemonGo-DesktopMap/wiki/Config.INI

**Comment by crystallinity** [comment]
Rayenous, are you using the 0.53/0.55 hash keys while running this? my understanding (though i haven't kept up on this project since it hasn't been updated in so long, and i had the same problem as described in here when i last tried it months ago) was that it has no captcha integration, so you have no way of knowing if your accounts have been captcha'd aside from checking the logs and looking for errors/failed scans; no built-in way of filling out the captchas when they occur; and no way of importing the new api key so you don't get tons of captchas/bans in the first place. have any of those issues been addressed somehow?

**Comment by Ang41187** [comment]
@Rayenous What's a good scan speed? I've only got 2 accounts right now

**Comment by MGRza** [comment]
Has anyone figured out how to make it work again?
